README.md

```diff
@@ -38,7 +38,7 @@ You will be learning and using the following hooks: `useReducer`, `useContext`,
 `useDebugValue`.
 
 1. `useReducer`: simple Counter
-2. `useReducer`: HTTP requests
+2. `useEffect`: HTTP requests
```

Comment:
Thanks for this! But actually we are focused on learning `useReducer` for exercise 2 because the `useEffect` is already there and you need to refactor the `useState`s to `useReducer` :)